### DISCUSSION TITLE
bilibili订阅重复推送

### DISCUSSION THREAD

**Discussion by Miuzarte** [Tsuk1ko/CQ-picfinder-robot#301] [问题求助]
挺怪的，订阅着明日方舟的没出事，另一个就一直在重复发
暂时没有日志
![Stitch_20220307_194543.png](https://user-images.githubusercontent.com/66856838/157028993-dba7032e-3b45-46b1-92c8-65d92d21cd56.png)

**Comment by Tsuk1ko** [comment]
很怪
